src/app/(routes)/customer/roadmap/[...slug]/page.tsx

```
@@ -175,225 +180,62 @@ const Page = () => {
-                        },
-                    ],
-                    premium: [],
-                },
-            },
-        };
-        return nodeDetails[nodeId] || nodeDetails['default'];
+        console.log('Fetching details for nodeId:', nodeId, 'with slug:', slug); // Debug log
+        console.log('Available frontendDetails keys:', Object.keys(frontendDetails)); // Debug all keys
+        switch (slug) {
+            case 'Frontend':
+                const detail = frontendDetails[nodeId] || frontendDetails['default'];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)